table-to-blt.mjs

```diff
@@ -37,15 +37,15 @@
 
 import { JSDOM } from 'jsdom';
 
-async function main(file, nbseats, electionname, sortballots = true, shownames = false) {
+export async function main(file, nbseats, electionname, sortballots = true, shownames = false) {
     if (isNaN(nbseats)) {
        throw new Error(`Required number of seats missing.`);
     }
     const dom = await JSDOM.fromFile(file);
-    generateBLT(dom.window.document, nbseats, electionname, sortballots, shownames);
+    return generateBLT(dom.window.document, nbseats, electionname, sortballots, shownames);
 }
 
-function generateBLT (document, nbseats, electionname, sortballots, shownames) {
+export function generateBLT (document, nbseats, electionname, sortballots, shownames) {
```

Comment:
not strictly necessary here, but this means that if this file gets put online (e.g. via github pages), it can be imported in a browser context and the generateBLT function used directly there.